### PR TITLE
Suggest using semver for plugin versioning in plugin.json

### LINKED ISSUE #71411: docs: recommend use of semver for plugin versioning
Just to provide guidance that this is a good way to indicate the nature of changes between versions at a glance for consumers. 

Should update `plugin.json` description(s) https://grafana.com/docs/grafana/latest/developers/plugins/metadata/

### PR DESCRIPTION
**Pull request by academo**
**What this PR does / why we need it**:

Closes https://github.com/grafana/grafana/issues/71411

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docusaurus/docs/metadata.md [hunk 1]
```diff
@@ -63,12 +63,12 @@ Plugin dependency. Used to display information about plugin dependencies in the
...
-| `version` | string | **Yes**  |                                                    |
+| Property  | Type   | Required | Description                                                                                                                                   |
+| --------- | ------ | -------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`      | string | **Yes**  |                                                                                                                                               |
+| `name`    | string | **Yes**  |                                                                                                                                               |
+| `type`    | string | **Yes**  | Possible values are: `app`, `datasource`, `panel`.                                                                                            |
+| `version` | string | **Yes**  | We suggest using [Semver](https://semver.org/) for your plugin versioning to alert users about potential changes, new features, or bug fixes. |
```
Comment (jackw) [inline docusaurus/docs/metadata.md:71]: nit: should we mention that a Semver range is best here?

Comment (academo) [reply docusaurus/docs/metadata.md:71]: @jackw this is the plugin version, not the grafana version. Unless I misunderstand your suggestion?

**Review by sympatheticmoose — APPROVED** [review]

File: docusaurus/docs/metadata.md (hunk 1 above)
Comment (jackw) [reply docusaurus/docs/metadata.md:71]: Ah ok. Sorry for the misunderstanding I'd assumed it was the Grafana version! 👍

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: docusaurus/docs/metadata.md (hunk 1 above)
Comment (josmperez) [inline docusaurus/docs/metadata.md:71]:
```suggestion
| `version` | string | **Yes**  | We suggest using [SemVer](https://semver.org/) for your plugin versioning to alert users about potential changes, new features, or bug fixes. |
```

**Review by josmperez — CHANGES_REQUESTED** [review]
One nit.

**Review by josmperez — APPROVED** [review]
LGTM.

**Pull request merged**